Gemfile

```diff
@@ -11,7 +11,7 @@ group :development do
   gem 'activerecord', "= #{ENV['activerecord'] || '4.1.8'}"
   gem 'em-mongo'
   gem 'bson_ext'
-  gem 'mysql2'
```

Comment:
I'm not sure why, but it failed to load an adapter https://travis-ci.org/igrigorik/em-synchrony/builds/102289375:

```
Specified 'em_mysql2' for database adapter, but the gem is not loaded. Add `gem 'mysql2'` to your Gemfile.
```

Probably, it is related to the issue https://github.com/rails/rails/issues/21544 (found it [here](https://stackoverflow.com/questions/32457657/rails-4-gemloaderror-specified-mysql2-for-database-adapter-but-the-gem-i)). This fix https://github.com/rails/rails/commit/5da5e3772c32593ecf2f27b8865e81dcbe3af692 hasn't been released yet.